src/sentry/static/sentry/app/components/tagDistributionMeter/index.jsx

```diff
@@ -38,6 +39,36 @@ export default class TagDistributionMeter extends React.Component {
     renderError: () => null,
   };
 
+  renderTitle() {
+    const {segments, totalValues, title} = this.props;
+
+    if (!Array.isArray(segments) || segments.length <= 0) {
+      return null;
+    }
+
+    if (!totalValues) {
```

Comment:
It's generally good practice to be very explicit about what you're checking. 

In this case, it would be clearer to use `totalValues === 0` to make it clear that you're checking against a number and not a boolean value.

Reply:
DONE